test/e2e/suite/nodeadm/nodeadm_test.go

```diff
@@ -98,7 +98,7 @@ var _ = Describe("Hybrid Nodes", func() {
 						k8sVersion = test.OverrideNodeK8sVersion
 					}
 
-					testNode := test.NewTestNode(ctx, instanceName, nodeName, k8sVersion, nodeOS, provider, e2e.Large)
+					testNode := test.NewTestNode(ctx, instanceName, nodeName, k8sVersion, nodeOS, provider, e2e.Large, false)
```

Comment:
This is what I meant by less readable code. I don't know what "false" means here.